Makefile.one

```diff
@@ -30,24 +31,77 @@ CMD_CAT ?= cat
 CMD_MD5 ?= md5sum
 CMD_OPA ?= opa # https://github.com/open-policy-agent/opa/releases/download/v0.33.1/opa_linux_amd64
 
-check_%:
-	@command -v $* >/dev/null || (echo "missing required tool $*" ; exit 1)
+.ONESHELL:
+.check_%:
+	@#
+	@command -v $* >/dev/null
+	if [ $$? -ne 0 ]; then
+		echo "missing required tool $*"
+		exit 1
+	else
+		touch $@ # avoid target rebuilds due to inexistent file
```

Comment:
By creating these temporary and ignored .dot files, we are basically telling $(MAKE) that this check has already been done and the file exists and hasn't changed. Without this, each target build depending on a `.check*` would be recompiled over and over because the dependency did not have an unchanged file to depend on.